ocaml/xapi/message_forwarding.ml

```diff
@@ -1399,8 +1415,8 @@ functor
             (Helpers.will_have_qemu ~__context ~self:vm) ;
           Xapi_network_sriov_helpers.reserve_sriov_vfs ~__context ~host ~vm
         with e ->
-          clear_scheduled_to_be_resident_on ~__context ~vm ;
-          clear_reserved_netsriov_vfs_on ~__context ~vm ;
+          clear_vif_reservations ~__context ~vm ;
```

Comment:
I understand you are probably just refacotring existing code. But why does `clear_vif_reservations` only appear in this place but not other places where `clear_reservations` is called?